SlamTests/MockQueueManager.swift

```diff
@@ -1,16 +1,25 @@
 import Slam
 import UIKit
 
-class MockQueueManager: QueueManager {
+public class MockQueueManager: QueueManager {
```

Comment:
This shouldn't need to be `public`. If you run into issues removing the `public`, I would double check that it is included in the test Target and not the main target.